typescript-express-reviews/package.json

```diff
@@ -19,7 +19,7 @@
     "dotenv": "16.0.1",
     "express": "4.x",
     "mongoose": "7.x",
-    "stargate-mongoose": "0.2.0-ALPHA-2"
+    "stargate-mongoose": "file:../../stargate-mongoose/stargate-mongoose-0.2.0-ALPHA-2.tgz"
```

Comment:
did you intend to include this? kind of looks like a local change for testing.

Reply:
Yep this is for local testing. Should not merge this branch until there's a compatible release of stargate-mongoose released.